create/templates/checkout_ui_extension/next-steps.txt

```diff
@@ -0,0 +1,11 @@
+
+🔭 > Your extension bundles are available at: 
+{{range .BundleUrls}}
+     * {{.}}
+{{end}}
+🔭 > If this is first time you are testing the extension,
+🔭 > please login to your development store first by visiting
+🔭 > https://{{.IntegrationContext.Store}}/password
+
+🔭 > To create a checkout and test your extension please visit the following URL:
```

Comment:
The url here is not quite right, there's no `/query` anymore. We should also just tell to click on the root url of the extension which already has internal logic to redirect to the right place.

Reply:
Do you mean, just list the URL that's after the `dev` parameter (minus the `/query`)?

Reply:
Sorry David, it had completely escaped me that we had setup redirect from the extension URL. Thanks for catching this Trish!